app/fhir_server/operations.py

```diff
@@ -24,11 +24,18 @@
 @routes.get("/Questionnaire/{id}/$assemble")
 async def assemble_handler(request: web.BaseRequest):
     client: AsyncFHIRClient = request.app["client"]
+    aidbox_client = request.aidbox_client
+
     questionnaire = (
         await client.resources("Questionnaire").search(_id=request.match_info["id"]).get()
     )
 
-    assembled_questionnaire_lazy = await assemble(client, to_first_class_extension(questionnaire))
+    def get_to_first_class_extension(fhir_resource):
```

Comment:
it should be `async def` instead of `def` because assemble accepts async function